openqa-maintenance.py

```diff
@@ -673,29 +498,41 @@ def calculate_repo_hash(self, repos):
             m.update(cs.text)
         return m.hexdigest()
 
-    def jobs_for_target(self, u):
-        s = u['settings'][0]
+    @staticmethod
+    def calculate_incidents(incidents):
+        """
+        get incidet numbers from SUSE:Maintenance:Test project
+        returns dict with openQA var name : string with numbers
+        """
+        l_incidents={}
+        for kind, prj in incidents.items():
+            url = "SUSE:Maintenance:Test:" + prj
+            iid = ','.join([x.split('.')[1] for x in osc.core.meta_get_packagelist("https://api.suse.de", url)])
```

Comment:
there is no need to hardcode the api url - at some point we might want to sync SLE and openSUSE maintenance ways